places/utcwebdev-chattanooga.md

```diff
@@ -0,0 +1,13 @@
+# Chattanooga, TN
+
+## Fun Facts
+- it is the scenic city
+
+## Places to Eat
+- Lupi's Pizza
+
+## Places to Go
+- Rock City
+
+## Places to Avoid
+- Soddy Daisy
```

Comment:
SD is a notoriously backward area in the county; it's basically the butt of jokes because of its name.